### PR TITLE
Support .env or -config env style files

### PR DESCRIPTION
**Pull request by pixie79**
Hi,

This supports #62, also when the next a small patch is merged into https://github.com/joho/godotenv/pull/53 then it will also support nested ENV settings within the config file as well.

Mark

### CODE REVIEW

**Comment by jaykeshur** [comment]
Thanks for the PR, please squash your commits

File: main.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ import (
 	"text/template"
 	"time"
 
+        "github.com/joho/godotenv"
```
Comment (jaykeshur) [inline main.go]: indentation

File: main.go [hunk 2]
```diff
@@ -131,6 +137,14 @@ func run(c *cli.Context) error {
 		return errors.New("no kubernetes resource files specified")
 	}
 
+        // Load Environment file overrides into the OS Environment Scope
+        if c.IsSet("config") {
+                err := godotenv.Load(c.String("config"))
+                if err != nil {
+                        log.Fatal("Error loading .env file")
```
Comment (jaykeshur) [inline main.go]: return error here (as above)

File: main.go [hunk 3]
```diff
@@ -63,6 +63,11 @@ func main() {
 			Usage:  "kubernetes auth `TOKEN`",
 			EnvVar: "KUBE_TOKEN,PLUGIN_KUBE_TOKEN",
 		},
+		cli.StringFlag{
+			Name:   "config",
+			Usage:  "Env file location",
+			EnvVar: "CONFIG_FILE",
```
Comment (gambol99) [inline main.go]: for consistency and use in a drone plugin can we plus PLUGIN_CONFIG_FILE in the EnvVar

**Pull request closed without merging**